.travis.yml

```diff
@@ -60,10 +60,10 @@ before_install:
 #Remove the old g++/gcc to ensure we're using the latest ones
  - if [ "$CXX" = "g++-4.9" -o "$CXX" = "g++-5" ]; then sudo rm /usr/bin/g++; sudo rm /usr/bin/gcc; fi
 #Install latest clang if we're compiling with clang
- - if [ "$CXX" = "clang++" ]; then sudo apt-get install -qq clang-3.7; fi
- - if [ "$CXX" = "clang++" ]; then export CXX="clang++-3.7" CC="clang-3.7"; fi
+ - if [ "$CXX" = "clang++" ]; then sudo apt-get install -qq clang-3.8; fi
+ - if [ "$CXX" = "clang++" ]; then export CXX="clang++-3.8" CC="clang-3.8"; fi
 #Remove the old clang to ensure we're using the latest ones
- - if [ "$CXX" = "clang++-3.7" ]; then sudo rm /usr/local/clang-3.4/bin/clang; sudo rm /usr/local/clang-3.4/bin/clang++; fi
+ - if [ "$CXX" = "clang++-3.8" ]; then sudo rm /usr/local/clang-3.4/bin/clang; sudo rm /usr/local/clang-3.4/bin/clang++; fi
```

Comment:
Ah, that's still a little way off due to libusb though, so I suspect I'll be in before @daveol .